test/parallel/test-child-process-stdio-big-write-end.js

```diff
@@ -29,7 +30,7 @@ function parent() {
 
   // Write until the buffer fills up.
   do {
-    var buf = new Buffer(BUFSIZE);
+    buf = new Buffer(BUFSIZE);
     buf.fill('.');
     sent += BUFSIZE;
   } while (child.stdin.write(buf));
```

Comment:
@silverwind Cool, I chose the former. It would be easier for the reviewers if we don't introduce logic changes like the later. Updated now :-)